generator/osm2type.cpp

```diff
@@ -461,6 +461,7 @@ string MatchCity(OsmElement const * p)
       {"taoyuan", {110.8471,28.4085,111.6109,29.4019}}, 
       {"tashkent", {69.12171, 41.163421, 69.476967, 41.398638}},
       {"tbilisi", {44.596922, 41.619315, 45.019694, 41.843421}},
+      {"tehran", {50.6575}},
```

Comment:
Expected 4 floats, like Rect

Reply:
Fixed. Should be 50.6575,35.353216,52.007904,35.974672